flask_jwt_extended/utils.py

```diff
@@ -245,7 +245,11 @@ def set_access_cookies(response, encoded_access_token, max_age=None):
                     JWT_SESSION_COOKIE option will be ignored.  Values should be
                     the number of seconds (as an integer).
     """
-    if not config.jwt_in_cookies:
+    if locations and "cookies" not in locations:
+        raise RuntimeWarning(
+            "set_access_cookies() called without " "locations configured to use cookies"
+        )
+    if not locations and not config.jwt_in_cookies:
         raise RuntimeWarning(
             "set_access_cookies() called without "
             "'JWT_TOKEN_LOCATION' configured to use cookies"
```

Comment:
I kind of think that ``set_access_cookies(locaions=['cookies'])` is a bit unwieldy. I think maybe we should just get rid of these warnings entirely, now that any endpoint could dynamically be set to cookies.